docs/providers.rst

```diff
@@ -236,6 +236,10 @@ Persistent Connection Providers
       connection and waiting for a response to be received from the listener task.
       Defaults to ``50.0``.
 
+    * ``request_information_cache_size`` is the size of the cache used to store
```

Comment:
This should probably get added to the signature of the class on line 225 above too.

Reply:
Ah, yep. Good eye 🙏🏼